### DISCUSSION TITLE
Table Format

### DISCUSSION THREAD

**Discussion by ashrafchowdury** [steven-tey/novel#40] [Q&A]
Hey @steven-tey 👋

I created the **Table format and implemented it in the editor**, now I want feedback from you to before making the pr-request

https://github.com/steven-tey/novel/assets/87828904/d5d36512-7399-4932-8f3f-e66555c2cfce

**I have added:**
* Adding & deleting Column
* Adding & deleting Row
* Delete Table
* A mune for the table

**Comment by yogeshbhutkar** [comment]
@steven-tey, @ashrafchowdury, by when can we expect this feature to roll out on the NPM package? I feel that this is a crucial enough feature and will be essential for a WYSIWYG editor.

**Comment by MohamedAliChaabani2030** [comment]
I love it i will be happy if i see this in this project

**Comment by supnim** [comment]
@steven-tey nudged for visibility, please may we have this in 🙏

**Comment by AlexanderBZ** [comment]
Nudging for visibility as well

**Comment by DMGithinji** [comment]
@steven-tey, @ashrafchowdury 
This would be big addition. When can we have this  🙏🏾
